instrumentation-genai/README.md

```diff
@@ -0,0 +1,6 @@
+
+| Instrumentation | Supported Packages | Metrics support | Semconv status |
```

Comment:
I'm curious why only one signal type "Metrics" is listed there. Would it make sense to list all of the signal types?

Reply:
This is the same format as the other instrumentations, no idea what the original intent but here we are just running the same thing on another dir.